### ISSUE TITLE
stop closing and let me settle this

### ISSUE THREAD

**Issue by btplayzxgit** [FlameDevV2/MagmaLauncher#8]
@radmanplays i didn't pirate any games, its just an unblocked games website and yes you are bullying/harassing us because you literally made a repo called "Erm.. what the skid?" to harass us. you call us not real people and not adults, but you are bullying us online, not very adult-like of you

**Comment by catfoolyou** [comment]
1) Your friend @FlameDevV2 skidded Resent, which is technically skidding Eagler (that you think is piracy)
2) "Erm.. what the skid?" was an issue, not a repo
3) @AR-DEV-1 needs to face the facts and quit closing every single issue

**Comment by radmanplays** [comment]
@btplayzxgit i agree with what @catfoolyou said bc the skidded resent client is **literally** in this repo that this issue was made in. https://magmalauncher.netlify.app/flameclient3.8
